grails-app/controllers/org/pih/warehouse/reporting/ReportController.groovy

```diff
@@ -603,7 +611,7 @@ class ReportController {
             render(contentType: "text/csv", text: CSVUtils.prependBomToCsvString(sw.toString()), encoding: "UTF-8")
         }
 
-        render(view: 'showInventoryByLocationReport', model: [command: command])
+        render(view: 'showInventoryByLocationReport', model: [command: command, includeCategoryChildren: includeCategoryChildren])
```

Comment:
As mentioned earlier, includeCategoryChildren should be included as a boolean property of the command class.